CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 **This is the changelog for the core Rust library**. There's a [separate changelog](./python/core/CHANGELOG.md) for the Python bindings, and there will be another for the JS bindings.
 
+## [0.3.0-beta.1] - 2024-08-23
+
+### Added
+
+- Indexed geometry arrays ([#443](https://github.com/geoarrow/geoarrow-rs/pull/443))
+- Parse row group stats out of GeoParquet ([#571](https://github.com/geoarrow/geoarrow-rs/pull/571))
+- Writing GeoParquet with GeoArrow encoding ([#583](https://github.com/geoarrow/geoarrow-rs/pull/583))
+- Async GeoParquet writing ([#587](https://github.com/geoarrow/geoarrow-rs/pull/587))
+- Bounding box queries on GeoParquet files ([#590](https://github.com/geoarrow/geoarrow-rs/pull/590))
```

Comment:
I think I'd really like to have full GeoParquet 1.1 support in the 0.3.0 release